.rubocop.yml

```diff
@@ -46,7 +46,7 @@ Metrics/ClassLength:
 
 # Avoid complex methods.
 Metrics/CyclomaticComplexity:
-  Max: 6
+  Max: 7
```

Comment:
Wouldn't it make more sense to ignore a specific method instead of increasing the threshold? Which method caused this?